docs/topics/settings.rst

```diff
@@ -1303,6 +1303,16 @@ Type of priority queue used by the scheduler. Another available type is
 domains in parallel. But currently ``scrapy.pqueues.DownloaderAwarePriorityQueue``
 does not work together with :setting:`CONCURRENT_REQUESTS_PER_IP`.
 
+.. setting:: SCHEDULER_DELAY_QUEUE
+
+SCHEDULER_DELAY_QUEUE
+---------------------
+Default: ``'scrapy.pqueues.ScrapyDelayedRequestsPriorityQueue'``
+
+Type of priority queue used by the scheduler for delayed requests. The default class is
+a sublcass of the ``scrapy.pqueues.ScrapyPriorityQueue``, so the chosen class must have
```

Comment:
```suggestion
a sublcass of the :class:`scrapy.pqueues.ScrapyPriorityQueue`, so the chosen class must have
```

It will probably link nowhere at the moment, but if we ever provide API documentation for it, then it will.

The same could be done for the default value above, since we support specifying this type of values as classes instead of strings since 2.4.0.